code/obj/item/table_rack_parts.dm

```diff
@@ -471,6 +471,13 @@ TYPEINFO(/obj/item/furniture_parts/table/glass)
 	icon_state = "scrapchair_parts"
 	furniture_type = /obj/stool/chair/wooden/scrap
 
+/obj/item/furniture_parts/wood_chair/industrial
+	name = "industrial chair parts"
+	desc = "An collection of rods and scaffolding that can be used to make an industrial chair."
+	icon_state = "ichair_parts"
+	furniture_type = /obj/stool/chair/wooden/industrial
```

Comment:
this doesn't look very wooden

Reply:
Neither does the scrap table, but it just so happens that "dining chairs" are all wooden. I was genuinely considering changing the name of the path, but figured that's excessive. But if this is asked for, I'll do it